src/test/kotlin/com/vanniktech/code/quality/tools/CodeQualityToolsPluginDetektTest.kt

```diff
@@ -11,67 +11,21 @@ import java.io.File
 class CodeQualityToolsPluginDetektTest {
   @get:Rule val testProjectDir = TemporaryFolder()
 
+  private val testCode = "package com.vanniktech.test;\nfun foo(param: Int) = param * param\n"
+  private val testPath = "src/main/kotlin/com/vanniktech/test/Foo.kt"
+
   @Test fun success() {
     Roboter(testProjectDir)
         .withConfiguration("failFast: true")
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun successBreakingReportChangeRC9() {
-    Roboter(testProjectDir, version = "1.0.0.RC9")
-        .withConfiguration("failFast: true")
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun successBreakingReportChangeRC92() {
-    Roboter(testProjectDir, version = "1.0.0.RC9.2")
-        .withConfiguration("failFast: true")
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
+        .withKotlinFile(testPath, testCode)
         .succeeds()
   }
 
-  @Test fun successBreakingReportChangeRC10() {
-    Roboter(testProjectDir, version = "1.0.0-RC10")
-        .withConfiguration("failFast: true")
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun worksWithRC11() {
-    Roboter(testProjectDir, version = "1.0.0-RC11")
-        .withConfiguration("failFast: true")
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun worksWithRC12() {
-    Roboter(testProjectDir, version = "1.0.0-RC12")
-        .withConfiguration("failFast: true")
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun worksWithRC13() {
-    Roboter(testProjectDir, version = "1.0.0-RC13")
-        .withConfiguration("") // Fail Fast is configured via the CLI parameter.
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun worksWithRC14() {
-    Roboter(testProjectDir, version = "1.0.0-RC14")
-        .withConfiguration("") // Fail Fast is configured via the CLI parameter.
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "fun foo(param: Int) = param * param\n")
-        .succeeds()
-  }
-
-  @Test fun worksWithRC15() {
-    Roboter(testProjectDir, version = "1.0.0-RC15")
-        .withConfiguration("") // Fail Fast is configured via the CLI parameter.
-        .withKotlinFile("src/main/kotlin/com/vanniktech/test/Foo.kt", "package com.vanniktech.test;\nfun foo(param: Int) = param * param\n")
-        .succeeds()
+  @Test fun worksWithV1_0_0() {
```

Comment:
Just:

```suggestion
  @Test fun works() {
```

Let's hope there won't be any further breaking changes! :D